src/contents/all/components/knot/KnotInput/KnotInput.tsx

```diff
@@ -2,13 +2,17 @@ import React, { useState, useRef, useEffect } from 'react';
 import { Timeline } from "antd";
 import RichTextEditor from 'react-rte/lib/RichTextEditor';
 
+import { User, Knot } from '../../../global/types';
+
 import KeyboardShortcut from '../../common/KeyboardShortcut/KeyboardShortcut'
 
 import "../Knot/Knot.scss"
 import "./KnotInput.scss";
 
 interface Props {
   createKnot: (bodyHtml: string, bodyText: string) => void
+  deleteKnot: any
```

Comment:
Might be able to refactor and put createKnot and editKnot here directly. deleteKnot would most likely go in `Knot.tsx` since it doesn't deal with RichTextEditor and is just a separate button in the Knot